storage/object/reference-content/cname-record.mdx

```diff
@@ -0,0 +1,89 @@
+
+## When and why do I need to create a CNAME record for Edge Services?
+
+When you enable Edge Services, initially your bucket content is served through the standard Edge Services endpoint, e.g. `bucket-name.svc.edge.scw.cloud`. If you do not want to customize the standard Edge Services endpoint, you do not need to worry about CNAME records.
+
+However, if you choose to [customize your Edge Services endpoint with your own subdomain](/storage/object/how-to/get-started-edge-services/#how-to-configure-a-custom-domain), you are prompted, as part of the process, to create a CNAME record to point your subdomain to the Edge Services endpoint for your bucket. 
+
+     <Lightbox src="scaleway-edge-services-configure-domain.webp" alt="" />
+
+## How to create a CNAME record
+
+Log into your domain provider, and locate the DNS settings for your domain. Create a new CNAME record pointing your subdomain to the Edge Services endpoint for your bucket. This endpoint will be displayed in the Scaleway console.
+
+The interface used by different domain providers varies, but creating your CNAME record may look like one of the following examples:
+
+| Record                                     | Destination                              |
+|--------------------------------------------|------------------------------------------|
+| `my-chosen-subdomain.beautiful-domain.com` | `bucket-name.svc.edge.scw.cloud.`         |
+
+| Subdomain                                  | Target host                                    |
+|--------------------------------------------|------------------------------------------------|
+| `my-chosen-subdomain`                      | `bucket-name.svc.edge.scw.cloud.`               |
+
+| Host record                                | Points to                                      |
+|--------------------------------------------|------------------------------------------------|
+| `my-chosen-subdomain`                      | `bucket-name.svc.edge.scw.cloud.`               |
+
+| `my-chosen-subdomain`                      | Record Type     | Value                                          |
+|--------------------------------------------|-----------------|------------------------------------------------|
+| `@`                                        | `CNAME`         | is an alias of `bucket-name.svc.edge.scw.cloud.`|
+
+<Message type="tip">
+The trailing dot at the end of the target endpoint (`bucket-name.svc.edge.scw.cloud.`) is implicitly added by some domain and DNS providers, and must be explicitly added for others. Check with yours whether the dot is necessary.
+</Message>
+
+You may also see a `TTL`field, which stands for **T**ime **T**o **L**ive.  This tells the DNS resolver how long it can cache this record, before it must re-check the original source in case something has changed. TTL is measured in seconds, and the default value is usually 12 hours (43200 seconds) or 24 hours (86400 seconds). 
```

Comment:
```suggestion
You may also see a `TTL` field, which stands for **T**ime **T**o **L**ive.  This tells the DNS resolver how long it can cache this record, before it must re-check the original source in case something has changed. TTL is measured in seconds, and the default value is usually 12 hours (43200 seconds) or 24 hours (86400 seconds). 
```